CONTRIBUTING.md

```diff
@@ -434,7 +434,7 @@ For people new to Rust, and just starting to contribute, or even for
 more seasoned developers, some useful places to look for information
 are:
 
-* The [rustc guide] contains information about how various parts of the compiler work and how to contribute to the compiler
+* The [rustc dev guide] contains information about how various parts of the compiler work and how to contribute to the compiler
```

Comment:
Makes me wonder if we should decide whether to use `rustc-dev-guide` or `rustc dev guide` and stick to one of them, or if it is completely insignificant

Reply:
well I was just naively translating things, unsure if it's significant but I guess we can discuss on a separate PR.